subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -170,6 +170,158 @@ config BT_TBS_CLIENT_MAX_TBS_INSTANCES
 	  Sets the maximum number of Telephone Bearer Service (TBS)
 	  instances to setup and use.
 
+config BT_TBS_CLIENT_BEARER_PROVIDER_NAME
+	bool "Read Bearer Provider Name Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Provider Name
+	  procedure.
+
+config BT_TBS_CLIENT_BEARER_UCI
+	bool "Read Bearer UCI Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer UCI procedure.
+
+config BT_TBS_CLIENT_BEARER_TECHNOLOGY
+	bool "Read Bearer Technology Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Technology procedure.
+
+config BT_TBS_CLIENT_BEARER_URI_SCHEMES_SUPPORTED_LIST
+	bool "Read Bearer URI Schemes Supported List Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer URI Schemes Supported
+	  List procedure.
+
+config BT_TBS_CLIENT_BEARER_SIGNAL_STRENGTH
+	bool "Read Bearer Signal Strength Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Signal Strength
+	  procedure.
+
+config BT_TBS_CLIENT_READ_BEARER_SIGNAL_INTERVAL
+	bool "Read Bearer Signal Strength Reporting Interval Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Signal Strength
+	  Reporting Interval procedure.
+
+config BT_TBS_CLIENT_SET_BEARER_SIGNAL_INTERVAL
+	bool "Set Bearer Signal Strength Reporting Interval Procedure"
+	default y
+	help
+	  This option enables support for the Set Bearer Signal Strength
+	  Reporting Interval procedure.
+
+config BT_TBS_CLIENT_BEARER_LIST_CURRENT_CALLS
+	bool "Read Bearer List Current Calls Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer List Current Calls
+	  procedure.
+
+config BT_TBS_CLIENT_CCID
+	bool "Read Content Control ID Procedure"
+	default y
+	help
+	  This option enables support for the Read Content Control ID
+	  procedure.
+
+config BT_TBS_CLIENT_INCOMING_URI
+	bool "Read Incoming Call Target Bearer URI Procedure"
+	default y
+	help
+	  This option enables support for the Read Incoming Call Target Bearer
+	  URI procedure.
+
+config BT_TBS_CLIENT_STATUS_FLAGS
+	bool "Read Status Flags Procedure"
+	default y
+	help
+	  This option enables support for the Read Status Flags Procedure
+	  procedure.
+
+config BT_TBS_CLIENT_CP_PROCEDURES
+	bool
+	default y
+	depends on (BT_TBS_CLIENT_CP_ACCEPT_CALL) || \
+	BT_TBS_CLIENT_CP_TERMINATE_CALL || \
+	BT_TBS_CLIENT_CP_HOLD_CALL || \
+	BT_TBS_CLIENT_CP_RETRIEVE_CALL || \
+	BT_TBS_CLIENT_CP_ORIGINATE_CALL || \
+	BT_TBS_CLIENT_CP_JOIN_CALLS
+	help
+	  This hidden option indicates that there are at least one control
+	  point procedure available.
+	  When this option is disabled it indicates that the control point can
+	  be optimized away.
+
+config BT_TBS_CLIENT_CP_ACCEPT_CALL
+	bool "Call Control Point Answer Incoming Call Procedure"
+	default y
+	help
+	  This option enables support for the Call Control Point Answer
+	  Incoming Call procedure.
```

Comment:
Seems to be a mismatch between the terms "accept" and "answer" - While semantically they probably mean the same, we should use the same term in the Kconfig name and the description IMO